src/modules/crud.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-redeclare */
+/* eslint-disable block-scoped-var */
+/* eslint-disable no-var */
+/* eslint-disable vars-on-top */
+const list = document.getElementById('todos-list');
+const addInput = document.getElementById('todo-input');
+
+export const createTodo = () => {
```

Comment:
- [ ] There are no indexes associated with task entries. Kindly ensure that new tasks have the property completed **set to false by default** and the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). This should be reflected in your local storage